packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -636,4 +636,8 @@
         <label>Erreur fatale : Impossible d'initialiser le composant {{0}}.</label>
         <name>quantic_UnableToInitializeComponent</name>
     </customLabels>
+        <customLabels>
+        <label>Vu par le client</label>
```

Comment:
Hey @cmartel-coveo 
I wanted to get your opinion on this translation.

<img width="688" alt="ViewedByCustomerBAdge" src="https://github.com/coveo/ui-kit/assets/86681870/f2eda842-61d1-4cae-bb08-c27063cb8f1e">

This the viewed by customer badge component, it will displayed on the results that were already consulted by the user.

Do you think that the translated label "Vu par le client" is good enough?